pkg/medusa/reconcile.go

```diff
@@ -9,25 +9,37 @@ import (
 	k8ss "github.com/k8ssandra/k8ssandra-operator/apis/k8ssandra/v1alpha1"
```

Comment:
Could we please split this file up, it doesn't have great cohesion and 'reconcile' is too broad a topic for a single file.

Reply:
This appears to be OK now.